src/AuthSamples/samples/Options.MultiTenant/Options.MultiTenant.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore" Version="$(MicrosoftAspNetCorePackageVersion)" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.Google" Version="$(MicrosoftAspNetCoreAuthenticationGooglePackageVersion)" />
```

Comment:
Google?

Reply:
@natemcmaster what's the new process to update the dependencies.props for AuthSamples?  We are using the google reference to pull in the dependency right now, we just need the MicrosoftAspNetCoreAuthenticationPackageVersion

Reply:
Update manually, or wait for @jkotalik who is starting on #4246 for auth samples soon.